CHANGELOG.md

```diff
@@ -23,14 +26,14 @@
 - Deduce the docker registry\_url from repository [\#37](https://github.com/inviqa/harness-go/pull/37) ([andytson-inviqa](https://github.com/andytson-inviqa))
 - Disable k8s service links to reduce unused environment variables [\#36](https://github.com/inviqa/harness-go/pull/36) ([andytson-inviqa](https://github.com/andytson-inviqa))
 - Make versions for DB image attributes that can be easily overrided [\#33](https://github.com/inviqa/harness-go/pull/33) ([jameshalsall](https://github.com/jameshalsall))
+- Replace prometheus annotations with operator podmonitor [\#29](https://github.com/inviqa/harness-go/pull/29) ([andytson-inviqa](https://github.com/andytson-inviqa))
 
 ## [0.4.0](https://github.com/inviqa/harness-go/tree/0.4.0) (2020-04-22)
 
 [Full Changelog](https://github.com/inviqa/harness-go/compare/0.3.0...0.4.0)
 
 **Merged pull requests:**
 
-- Replace prometheus annotations with operator podmonitor [\#29](https://github.com/inviqa/harness-go/pull/29) ([andytson-inviqa](https://github.com/andytson-inviqa))
```

Comment:
This does seem to be in 0.4.0

Reply:
argh, just merged this after Dave's approval, that is odd though, it seems to move things around randomly when running the changelog generator? 🤔